lib/android/nit_activity.nit

```diff
@@ -279,6 +286,11 @@ class Activity
 	# Notification from Android, the current device configuration has changed
 	fun on_configuration_changed do end
 
+	# The back key has been pressed
+	#
+	# Return `true` if the event has been handled.
+	fun on_back_pressed: Bool do return false
```

Comment:
The doc could be clearer... It's the "stardard" way of doing this in the app.nit framework.  The return value is passed to the underlying Java code. Refinements can return true to intercept the event, but the default returns false and let the OS do its thing. In this case it's handle the back button and close the app.